### PR TITLE
Add 'Logger.log_text' method.

### PR DESCRIPTION
**Pull request by tseaver**
Doesn't yet expose any of the optional parameters (see #1566).

### CODE REVIEW

**Comment by tseaver** [comment]
@dhermes PTAL.  I will rebase #1568 and assign to you after this one lands.

File: gcloud/logging/logger.py [hunk 1]
```diff
@@ -41,3 +41,49 @@ def client(self):
     def project(self):
         """Project bound to the logger."""
         return self._client.project
+
+    @property
+    def full_name(self):
+        """Fully-qualified name used in logging APIs"""
+        return 'projects/%s/logs/%s' % (self.project, self.name)
+
+    def _require_client(self, client):
```
Comment (dhermes) [inline gcloud/logging/logger.py]: `require` isn't quite the right word if we aren't going to error out in the absence?

**Comment by dhermes** [comment]
Done with review.

File: gcloud/logging/logger.py (hunk 1 above)
Comment (tseaver) [inline gcloud/logging/logger.py]: At this point `_require_client` is ubiquitous in our codebase.

**Comment by tseaver** [comment]
Is that an `LGTM`?

**Comment by dhermes** [comment]
Nope

**Comment by tseaver** [comment]
What issue remains, besides `_require_client`, which we use all over the place?

**Comment by tseaver** [comment]
```bash
$ git grep -l _require_client
gcloud/bigquery/dataset.py
gcloud/bigquery/job.py
gcloud/bigquery/query.py
gcloud/bigquery/table.py
gcloud/dns/changes.py
gcloud/dns/zone.py
gcloud/pubsub/subscription.py
gcloud/pubsub/topic.py
gcloud/resource_manager/project.py
gcloud/search/document.py
gcloud/storage/_helpers.py
gcloud/storage/acl.py
gcloud/storage/blob.py
gcloud/storage/bucket.py
```

**Comment by dhermes** [comment]
LGTM. Sorry for the hold up, I was very distracted at that moment so didn't want to jump the gun.

**Comment by tseaver** [comment]
No worries!

**Pull request merged**